### PR TITLE
PORTALS-2282

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Demo: 
https://user-images.githubusercontent.com/1864447/185213477-93d51d2c-d415-43cf-b293-2715f37b2d95.mov

File: src/lib/containers/table/SynapseTable.tsx [hunk 1]
```diff
@@ -411,22 +407,6 @@ export default class SynapseTable extends React.Component<
...
-              >
-                <TotalQueryResults
-                  style={{ fontSize: 15 }}
-                  frontText={'Showing'}
-                  applyChanges={(newFacets: FacetColumnRequest[]) =>
-                    this.applyChangesFromQueryFilter(newFacets)
-                  }
-                />
-              </div>
-            )}
```
Comment (nickgros) [inline src/lib/containers/table/SynapseTable.tsx:429]: I believe with these changes, TotalQueryResults can no longer be shown in a StandaloneQueryWrapper if no TopLevelControls is shown. Not sure if this is unintended, and not sure if it's a big deal--depends on how StandaloneQueryWrapper is typically configured in portals/SWC.

**Review by nickgros — APPROVED** [review]

**Comment by jay-hodgson** [comment]
Design was just changed.
Screen Shot 2022-08-17 at 1 28 36 PM

**Comment by jay-hodgson** [comment]
https://user-images.githubusercontent.com/1864447/185473008-0e738788-4689-493b-b642-7b962bb36788.mov

File: src/lib/containers/TotalQueryResults.tsx [hunk 2]
```diff
@@ -327,12 +327,16 @@ const TotalQueryResults: FunctionComponent<TotalQueryResultsProps> = ({
...
+  const hasFilters =
+    facetsWithSelection.length > 0 ||
+    (searchSelectionCriteriaPillProps &&
+      searchSelectionCriteriaPillProps.length > 0)
   if (error) {
     return <></>
   }
   return (
     <div
-      className={`TotalQueryResults ${showNotch ? 'notch-down' : ''} ${
```
Comment (nickgros) [inline src/lib/containers/TotalQueryResults.tsx:335]: Remove corresponding CSS class

File: src/lib/containers/table/TopLevelControls.tsx [hunk 3]
```diff
@@ -156,18 +149,31 @@ const TopLevelControls = (props: TopLevelControlsProps) => {
...
-                  endText={hasFacetsOrFilters ? 'Filtered By' : ''}
-                  showNotch={showNotch}
+                <QueryCount
+                  sql={getInitQueryRequest().query.sql}
+                  parens={true}
                 />
               )}
-              {!hideQueryCount && entity && (
-                <MissingQueryResultsWarning entity={entity} />
-              )}
```
Comment (nickgros) [inline src/lib/containers/table/TopLevelControls.tsx:170]: Where is MissingQueryResultsWarning going to go? Didn't see where it was relocated, if it was

File: src/__tests__/lib/utils/hooks/useGetInfoFromIds.test.tsx [hunk 4]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import useGetInfoFromIds, {
   UseGetInfoFromIdsProps,
 } from '../../../../lib/utils/hooks/useGetInfoFromIds'
-import { act } from 'react-dom/test-utils'
+import { act } from '@testing-library/react'
```
Comment (nickgros) [inline src/__tests__/lib/utils/hooks/useGetInfoFromIds.test.tsx:5]: Not actually using React Testing Library here, so the original `act`was correct

File: src/lib/containers/table/TopLevelControls.tsx (hunk 3 above)
Comment (jay-hodgson) [reply src/lib/containers/table/TopLevelControls.tsx]: Missed that in the revert (which is why the build failed too!).  Brought it back

File: src/lib/containers/TotalQueryResults.tsx (hunk 2 above)
Comment (jay-hodgson) [reply src/lib/containers/TotalQueryResults.tsx:335]: Removed

File: src/__tests__/lib/utils/hooks/useGetInfoFromIds.test.tsx (hunk 4 above)
Comment (jay-hodgson) [reply src/__tests__/lib/utils/hooks/useGetInfoFromIds.test.tsx]: Reverted

**Pull request merged**